README.md

```
@@ -33,12 +33,12 @@ Every simulator has both 2-D and 3-D implementations.
 
 ## Quick Start
 
-You will need CMake to build the code. If you're using Windows, you need Visual Studio 2017 in addition to CMake.
+You will need CMake and [vcpkg](https://github.com/microsoft/vcpkg) to build the code. Set `VCPKG_ROOT` to your vcpkg checkout; CubbyFlow's manifest installs the required libraries during CMake configuration. If you're using Windows, you also need Visual Studio.
 
 First, clone the code:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)